tokens/.style-dictionary/config.js

```diff
@@ -0,0 +1,51 @@
+const { inheritedVariablePrefix } = require( '../build/wikimedia-ui-base/config' );
+
+const StyleDictionary = require( 'style-dictionary' ).extend( {
+	include: ['node_modules/wikimedia-ui-base/tokens.json'],
+	source: ['properties/**/*.json'],
+	platforms: {
+		scss: {
+			transformGroup: 'scss',
+			buildPath: 'dist/',
+			files: [{
+				destination: '_variables.scss',
+				format: 'scss/variables',
+				filter: 'removePrefixedVars'
```

Comment:
`removePrefixedVars` sounds rather generic but by the time it gets registered it is highly opinionated already (looking for the wmf-prefix). Maybe we should reflect this fact in its name to make the config more self-explanatory.

```suggestion
				filter: 'removeBaseTokensByWMF'
```

Reply:
(changed my mind about the name and edited)